packages/block-editor-tools/src/hooks/use-terms/README.md

```diff
@@ -20,17 +20,17 @@ const MyComponent = ({
 
   return (
     <SelectControl
-      label={__('My Terms', 'board-connect')}
```

Comment:
Good call here, although I'd still recommend setting `alley-scripts` as the example namespace, or something suitably generic like `your-textdomain-here`